x-pack/plugins/maps/public/lens/choropleth_chart/expression_renderer.tsx

```diff
@@ -48,6 +78,20 @@ export function getExpressionRenderer(coreSetup: CoreSetup<MapsPluginStartDepend
         // ignore error, lack of EMS file layers will be surfaced in dimension editor
       }
 
+      const renderComplete = () => {
+        const executionContext = handlers.getExecutionContext();
+        const containerType = extractContainerType(executionContext);
+        const visualizationType = extractVisualizationType(executionContext);
+
+        if (containerType && visualizationType) {
+          plugins.usageCollection?.reportUiCounter(containerType, METRIC_TYPE.COUNT, [
+            `render_${visualizationType}_map`,
```

Comment:
Can you use `regionmap` instead of `map`. In the future, additional map type visualizations may be added.